### PR TITLE
Add L1 matching by quality + other minor changes

### PR DESCRIPTION
**Pull request by battibass**
This PR:
1. adds to TnP L1 trigger information matched using a "by quality" sorting (*)
2. updates the list of files used in tp_from_aod_Data.py to a list of files available in EOS
3. adds .root files to .gitignore

(*) to work, the by quality matching relies on the following branch: https://github.com/battibass/cmssw/tree/stage2_L1_matching_80X

### CODE REVIEW

File: test/zmumu/tp_from_aod_MC.py [hunk 1]
```diff
@@ -77,7 +77,8 @@
 process.muonMatchHLTL3.maxDeltaR = 0.1
 from MuonAnalysis.MuonAssociators.patMuonsWithTrigger_cff import *
 changeRecoMuonInput(process, "mergedMuons")
-#useExtendedL1Match(process) #MM no idea what the sequence did, not available since git migration
+#useL1Stage2Candidates(process)
+#appendL1MatchingAlgo(process)
```
Comment (gpetruc) [inline test/zmumu/tp_from_aod_MC.py:81]: shouldn't `appendL1MatchingAlgo` be uncommented in MC?

**Review by gpetruc — COMMENTED** [review]
Looks ok to me.
I'd suggest also making the PRs for CMSSW in 80X and 81X

**Comment by battibass** [comment]
81X and 80X PRs done: https://github.com/cms-sw/cmssw/pull/16028 https://github.com/cms-sw/cmssw/pull/16029

File: test/zmumu/tp_from_aod_MC.py (hunk 1 above)
Comment (battibass) [reply test/zmumu/tp_from_aod_MC.py:81]: Also useL1Stage2Candidates in theory, though, as I don't know what @HuguesBrun  and @calderona plans are for updating the production of TnP ntuples in MC, I left them commented for now. I can uncomment if they want me to do that.

Comment (HuguesBrun) [reply test/zmumu/tp_from_aod_MC.py:81]: As the new MC will have the new L1, these 2 lines can be uncommented I guess...
@battibass are you fine with doing it in this PR ?

Comment (battibass) [reply test/zmumu/tp_from_aod_MC.py:81]: Hi, as I'm going to be away for a few days (disappearing ~ now), and I think you are in a hurry for setting up a new version of TnP production for data, I would suggest I look for a proper sample (with reHLT) and I setup everything properly in a future PR, ok?

Comment (HuguesBrun) [reply test/zmumu/tp_from_aod_MC.py:81]: yes, we will do a PR in few day before submitting the reHLT DY sample

**Pull request merged**